pages/partners.js

```diff
@@ -291,6 +291,9 @@ const Partners = () => (
     <SyntaxHighlighter language="bash" style={docco}>
       {CURL_SENCROP_PARTNER_USER_RETRIEVAL}
     </SyntaxHighlighter>
+    <p>
+      Then your <code>{"<PARTNER_ID>"}</code> will be in the <code>{'organisationId'}</code> field.
```

Comment:
I think the `{''}` part of the `organisationId` mention is unecessary. The reason why it makes sense for the `<PARTNER_ID>` part is that the `<` `>` symbols would be recognised as JSX/HTML markup.